update alt text on images on studio (#5809)
aws-amplify/docs: src/pages/console/data/data-model.mdx

The Studio data model designer provides a visual way to define your app's data model, relationships, and authorization rules. Studio generates a `schema.graphql` GraphQL Transform for the data model you create. To learn more about how the GraphQl transform works, see [GraphQL Transform Overview](/cli/graphql-transformer/overview).

> ![Screen recording on process to add a data model in data modelling page](/images/console/datamodel.gif)

Reviewer: ```suggestion
![Screen recording of the process process of adding a data model in data modelling page](/images/console/datamodel.gif)
```
